common/views/components/PrototypeSearchFilters/PrototypeSearchFiltersMobile.tsx

```diff
@@ -166,6 +164,7 @@ const SearchFiltersMobile = ({
   locationsTypeInUrlArray,
   imagesColor,
   aggregations,
+  filtersToShow,
```

Comment:
presume this is already being passed in?

Reply:
yeah